tanner/utils/php_sandbox_helper.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import asyncio
+import aiohttp
+from tanner import config
+
+
+class PHPSandboxHelper:
+    def __init__(self, loop):
+        self.logger = logging.getLogger('tanner.php_sandbox_helper.PHPSandboxHelper')
+        self._loop = loop if loop is not None else asyncio.get_event_loop()
+
+    async def injection_result(self, code):
```

Comment:
It's a general method to get result of php code so I guess give it a better name.